modules/op-running-pipeline-run-and-task-run-with-custom-scc-and-service-account.adoc

```diff
@@ -0,0 +1,121 @@
+[id=op-running-pipeline-run-and-task-run-with-custom-scc-and-service-account_{context}]
```

Comment:
Check all IDs are formatted properly for these files (reference: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-in-module-files)